components/authz-service/storage/v2/memstore/memstore.go

```diff
@@ -372,7 +372,7 @@ func (*State) FetchAppliedRulesByProjectIDs(ctx context.Context) (map[string][]*
 	return nil, nil
 }
 
-func (*State) ErrIfMissingProjects(ctx context.Context, projectIDs []string) error {
+func (*State) EnsureNoProjectsMissing(ctx context.Context, projectIDs []string) error {
```

Comment:
[nit] Could be `func (*State) EnsureNoProjectsMissing(context.Context, []string) error {` but doesn't matter 😉

Reply:
I'm hoping this code goes away soon regardless.